src/providers/key/key.ts

```diff
@@ -276,4 +278,11 @@ export class KeyProvider {
 
     return key.sign(rootPath, txp, password);
   }
+
+  public match(key1, key2) {
```

Comment:
It should be a private function? I'd change the name by another non-reserved word

Reply:
Is public I use it here: https://github.com/bitpay/copay/pull/10696/files#diff-a287f4baee716e25e5027889f0b17d93R903

But, I change the function name thanks!